lib/awful/hotkeys_popup/widget.lua

```diff
@@ -570,47 +570,40 @@ function widget.new(args)
         end
         table.insert(pages, columns)
 
-        local mywibox = wibox({
+        local mypopup = awful.popup {
+            widget = pages[1],
             ontop = true,
             bg=self.bg,
             fg=self.fg,
             opacity = self.opacity,
             border_width = self.border_width,
             border_color = self.border_color,
             shape = self.shape,
-        })
+            placement = awful.placement.centered,
+            minimum_width = wibox_width,
+            minimum_height = wibox_height,
+        }
+
         local widget_obj = {
             current_page = 1,
-            wibox = mywibox,
-        }
-        mywibox:geometry({
-            x = wa.x + math.floor((wa.width - wibox_width - self.border_width*2) / 2),
-            y = wa.y + math.floor((wa.height - wibox_height - self.border_width*2) / 2),
-            width = wibox_width,
-            height = wibox_height,
-        })
-        mywibox:set_widget(pages[1])
-
-        mywibox.buttons = {
-            awful.button({ }, 1, function () widget_obj:hide() end),
-            awful.button({ }, 3, function () widget_obj:hide() end)
```

Comment:
why you removed the mouse bindings?

Reply:
Handled already in [`awful.popup`](https://github.com/awesomeWM/awesome/blob/master/lib/awful/popup.lua#L282`)

Reply:
Did a test drive to verify and I was wrong.. Doing a fixup commit real quick